documentation/src/main/asciidoc/topics/proc_adding_maven_repository.adoc

```diff
@@ -0,0 +1,50 @@
+[id='mvn_settings']
+//Community
+ifndef::productized[]
+= Using {brandname} Maven Settings
```

Comment:
you don't need this. Infinispan is available in maven central.
You only need to change the maven settings if you are building Infinispan itself.